compiler-rt/lib/builtins/aarch64/sme-libc-routines.c

```diff
@@ -0,0 +1,87 @@
+#include <stdlib.h>
```

Comment:
I've #79454 to switch this out for `stddef.h` which is provided by the compiler resource headers rather than the libc (since this is inside builtins).